src/backend/distributed/planner/multi_master_planner.c

```diff
@@ -244,6 +250,71 @@ BuildSelectStatement(Query *masterQuery, List *masterTargetList, CustomScan *rem
 }
 
 
+/*
+ * QueryContainsAggregateWithHLL returns true if the query has an hll aggregate
+ * function in it's target list.
+ */
+static bool
+QueryContainsAggregateWithHLL(Query *query)
```

Comment:
Our function definition logic is that
1 - we declare the function
2 - we use the function 
3 - we provide the implementation of the function at the first available place.

Thus function definition order should look like
```
BuildAggregatePlan
HasDistinctAggregate
UseGroupAggregateWithHLL
QueryContainsAggregateWithHLL
```